web/src/reducers/index.js

```diff
@@ -49,6 +49,7 @@ const initialApplicationState = {
   solarColorbarValue: null,
   webGLSupported: true,
   windColorbarValue: null,
+  selectedTimeAggregate: 'day',
```

Comment:
Arguably this could be called 'selectedTimeRange' since the aggregate is 'hourly'. We could wait for another PR as we will touch this again soon.

Reply:
Yeah, I think this is fine for now and then we'll think more about it later on